subprojects/scala/src/integTest/groovy/org/gradle/scala/ScalaCompilationFixture.groovy

```diff
@@ -101,6 +101,10 @@ class ScalaCompilationFixture {
 
             sourceCompatibility = '${sourceCompatibility}'
             targetCompatibility = '${sourceCompatibility}'
+
+            tasks.withType(AbstractScalaCompile) {
+                scalaCompileOptions.additionalParameters = ["-target:jvm-1.8"]
```

Comment:
❓ Why do we need `-target:jvm-1.8` here? Because the given Scala version cannot produce bytecode for the current JVM version? How did this work before?

Reply:
This was a demonstration of how the users can fix the breaking change when running Gradle with any JVM above 8.

However, these fixes for tests will not be required anymore (already reverted), since we will make the selection of the implicit target for Scala smarter by falling back to Java 8 if no toolchain was configured.